### PR TITLE
backport cpu mitigations

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: src/clients/firewall_stage1_proposal.rb [hunk 1]
```diff
@@ -130,17 +135,17 @@ def main
           raise "INTERNAL ERROR: unknown action '#{@chosen_link}' for proposal client"
         end
 
-        SuSEFirewallProposal.SetChangedByUser(true)
+        SuSEFirewallProposal.SetChangedByUser(true)
```
Comment (teclator) [inline src/clients/firewall_stage1_proposal.rb]:
```suggestion
        SuSEFirewallProposal.SetChangedByUser(true)
```

File: test/firewall_stage1_proposal_test.rb [hunk 2]
```diff
@@ -1,61 +0,0 @@
...
-require "yast"
-
-module Yast
-  extend Yast::I18n
-
-  import "SuSEFirewall4Network"
-  import "SuSEFirewallProposal"
-  import "WFM"
-
-  describe "FirewallStage1ProposalClient" do
```
Comment (teclator) [inline test/firewall_stage1_proposal_test.rb]: Deleting tests is not so nice :(

Comment (jreidinger) [reply test/firewall_stage1_proposal_test.rb]: yep, not so nice, but for pure clients it is hard to mock them and I need to mock that bootloader loading part as bootloader is not build requirement.

**Comment by teclator** [comment]
JFR: It basically backports https://github.com/yast/yast-firewall/pull/123, but in SLE-12-SP5 the first stage firewall proposal was in this repository instead. It has been unified in SLE-15 when adding firewalld based proposal.

**Comment by teclator** [comment]
LGTM once provided version bump & changelog.

**Pull request merged**